Cargo.lock

```diff
@@ -3342,6 +3342,25 @@ dependencies = [
 name = "rmc"
 version = "0.1.0"
 
+[[package]]
+name = "rmc-link-restrictions"
+version = "0.1.0"
+dependencies = [
+ "rmc_restrictions",
+ "rustc_data_structures",
```

Comment:
What are we using from here?

Reply:
The compiler's `FxHashMap`, not strictly necessary/could be replaced with a vanilla `HashMap`

Reply:
If easy, could do that here; if not, lets just do that as a cleanup PR